### ISSUE TITLE
Error: The terraform-provider-crowdstrike_v0.0.5 plugin crashed!

### ISSUE THREAD

**Issue by shiransiboni** [CrowdStrike/terraform-provider-crowdstrike#35]
When I tried to updated the prevention policies I got the fowling error: 
The thing is that the policies has been created in the UI but I can't made any changes 

![image](https://github.com/user-attachments/assets/c65fbc7f-ddb0-4cbe-a2d2-7209ccdb4b28)

**Comment by shiransiboni** [comment]
I verified and I have the right permissions in the API scope. 
I attached the one of the policies; the issue is that I got the error here but the policies added to the UI

![Screenshot 2024-07-30 at 10 14 28](https://github.com/user-attachments/assets/ebf984b8-aae9-48a2-87b0-72986e8dddd4)

**Comment by ffalor** [comment]
@shiransiboni the host groups `[""]` is causing the error. Removing that line or try `[]` should stop the error from happening. The error handling bug still exists and Ill have a fix out soon, but that will prevent your manifest from erroring.

**Comment by shiransiboni** [comment]
thanks! it works. but I come across with a new bug - I opened a new issue

**Comment by ffalor** [comment]
@shiransiboni glad to hear, I've responded to the other issue.

**Comment by ffalor** [comment]
#39 made it no long possible to provide an empty value for host group or rule groups. It also updated the error handling logic to no longer panic for other invalid host/rule groups.

**Comment by ffalor** [comment]
Fixed in version `v0.0.7`

```terraform
terraform {
  required_providers {
    crowdstrike = {
      source = "CrowdStrike/crowdstrike"
      version = "0.0.7"
    }
  }
}
```
